build.gradle

```diff
@@ -1,7 +1,16 @@
-import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
-
 buildscript {
-    ext.kotlinVersion = '1.4.21-2'
+    ext {
+        kotlinVersion = '1.4.21'
+        kotlinCoroutinesVersion = '1.4.2-native-mt'
```

Comment:
Are you sure we want Kotlin/Native here?

Reply:
Nop, should be 1.4.2.